packages/block-library/src/template-part/index.php

```diff
@@ -32,7 +32,18 @@ function render_block_core_template_part( $attributes ) {
 	if ( is_null( $content ) ) {
 		return 'Template Part Not Found';
 	}
-	return apply_filters( 'the_content', str_replace( ']]>', ']]&gt;', $content ) );
+
+	// Run through the actions that are typically taken on the_content.
+	$content = do_blocks( $content );
+	$content = wptexturize( $content );
+	$content = convert_smilies( $content );
+	$content = wpautop( $content );
+	$content = shortcode_unautop( $content );
+	$content = prepend_attachment( $content );
+	$content = wp_make_content_images_responsive( $content );
```

Comment:
I noticed that this triggers a warning on WordPress trunk 

<img width="1422" alt="Capture d’écran 2020-04-09 à 5 23 32 PM" src="https://user-images.githubusercontent.com/272444/78917619-0ae25480-7a87-11ea-81d8-4713e57b4c05.png">

We should probably update to use the latest behavior instead

Reply:
@youknowriad PR opened: https://github.com/WordPress/gutenberg/pull/21514